doc/source/rllib-algorithms.rst

```diff
@@ -887,3 +887,65 @@ Intrinsic rewards for each env-step are calculated by taking the euclidian dista
 ("forward" model).
 This allows the agent to explore areas of the environment, where the "forward" model still performs poorly (are not "understood" yet), whereas exploration to these areas will taper down after the agent has visited them
 often: The "forward" model will eventually get better at predicting these next latent vectors, which in turn will diminish the intrinsic rewards (decrease the euclidian distance between predicted and actual vectors).
+
+
+
+.. _RE3:
+
+RE3 (Random Encoders for Efficient Exploration)
+-----------------------------------------------
+
+|Tensorflow|
+`[paper] <https://arxiv.org/pdf/2102.09430.pdf>`__
+`[implementation] <https://github.com/ray-project/ray/blob/master/rllib/utils/exploration/random_encoder.py>`__
+
+Tuned examples:
```

Comment:
just "Examples:" instead of "Tuned examples:"?
RLlib actually has a tuned_examples folder that has a bunch of script that are "tuned".
since this one lives in examples/ folder, maybe it's less confusing to just say "Examples:"

Reply:
thanks, will update.